common/CMakeLists.txt

```diff
@@ -76,6 +81,8 @@ set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
 link_directories(${CMAKE_BINARY_DIR})
 
-ADD_SUBDIRECTORY (packages/tinyscheme)
-ADD_SUBDIRECTORY (interpreter)
+if(NOT UNTRUSTED_ONLY)
+	ADD_SUBDIRECTORY (packages/tinyscheme)
+	ADD_SUBDIRECTORY (interpreter)
+endif()
 ADD_SUBDIRECTORY (tests)
```

Comment:
you have to build these anyway. we need the tinyscheme shared library for testing.